docs/csharp/roslyn-sdk/source-generators-overview.md

```diff
@@ -72,6 +72,8 @@ In this guide, you'll explore the creation of a source generator using the <xref
 
     > [!TIP]
     > The source generator project needs to target the `netstandard2.0` TFM, otherwise it will not work.
+    > The source generator can be hosted under .NET Framework environment (in Visual Studio IDE or MSBuild CLI), or .NET 7.0+ environment (`dotnet` CLI, VS Code, or VS out-of-process).
+    > So, to support all the possible environments where a source generator can run in, it should be `netstandard2.0`.
```

Comment:
I still think the fundamental message here needs amendment; perhaps more so along the lines of:

```suggestion
    > [!TIP]
    > The source generator project should typically target the `netstandard2.0` TFM, as to support their consumption within .NET Framework environments including the Visual Studio IDE and MSBuild CLI.
```

Reply:
I'll let someone from the compiler team decide the best wording.

Reply:
Either way seems fine to me.